tools/ingest-converter/src/main/resources/ingest-shared.js

```diff
@@ -142,5 +142,44 @@ var IngestConverter = {
         return "if " + this.quote_string(tag) + " in [tags] {\n" +
                 on_failure_pipeline + "\n" +
                 "}";
+    },
+
+    get_elasticsearch_output: function () {
+        return this.fix_indent("output {\n" +
+            "elasticsearch {\n" +
+            "hosts => \"localhost\"\n" +
+            "}\n" +
+            "}");
+    },
+
+    get_stdin_input: function () {
+        return this.fix_indent("input {\n" +
+            "stdin {\n" +
+            "}\n" +
+            "}");
+    },
+
+    get_stdout_output: function () {
+        return this.fix_indent("output {\n" +
+            "stdout {\n" +
+            "codec => \"rubydebug\"\n" +
+            "}\n" +
+            "}");
+    },
+
+    get_newline: function () {
+        return "\n";
+    },
+
+    append_io_plugins: function(filters_pipeline, append_stdio) {
+        if (append_stdio === true) {
+            return IngestConverter.get_stdin_input() +
```

Comment:
@suyograo this is kinda weird, right? :D

Doesn't really matter but why not save the complication of the `get_newline` and just do

```js
return [IngestConverter.get_stdin_input(), filters_pipeline, IngestConverter.get_stdout_output()].join("\n");
```

? (same for the below use of this, could even dry up the `join("\n")` and just set up the array in the condition) :)